logs/worker-1-log

```diff
@@ -0,0 +1,431 @@
+[33mworker_1       |[0m Setting up cgroup
```

Comment:
I think these two log files accidentaly made it to the PR.

Reply:
Ah yes, whoops!